src/data/faq_de.json

```
@@ -335,6 +335,22 @@
+                            "<b>Warum wird der 2G+-Status nicht angezeigt, obwohl ich ein Impf- oder Genesenenzertifikat und einen negativen Test habe?</b>",
+                            "Der 2G+-Status ergibt sich aus der Kombination von Impf- und Testzertifikaten. Wenn die Daten innerhalb der Zertifikate, beispielsweise Name oder Geburtsdatum, nicht übereinstimmen, kann die Corona-Warn-App diese Zertifikate nicht zueinander zuordnen und den entsprechenden Status ermitteln. In diesem Fall zeigt die Corona-Warn-App Ihnen zwei Zertifikate in der Übersicht an, denen jeweils die Regeln 3G und 2G zugeordnet sind. Stellen Sie daher sicher, dass Ihr Name beim Ausstellen des Zertifikats korrekt und vollständige eingetragen wird, und wenden Sie sich an den Aussteller des Zertifikats, sollten Fehler beim Eintrag des Namens oder Geburtsdatums entstanden sein.",
+                            "Ein Testzertifikat bestätigt ein negatives Testergebnis offiziell in Form eines QR-Codes in der Corona-Warn-App. Sie können das offizielle Testzertifikat für PCR- und Schnelltests anfordern, wenn sie einen Test in der Corona-Warn-App registrieren. Dafür können sie den universellen QR-Code-Scanner in der Mitte der Registerkarte der CWA öffnen und den QR-Code zur Test-Registrierung scannen. Den erhalten sie entweder schon während der Buchung des Tests oder vor Ort beim Testen, sofern der Anbieter an die Corona-Warn-App angeschlossen ist. Nach dem Scannen des QR-Codes öffnet sich automatisch ein Fenster, in dem sie das Testzertifikat anfordern können. Mehr Informationen finden Sie in diesem Blog: <a href='../blog/2021-06-24-cwa-version-2-4' target='_blank'>Projektteam veröffentlicht Online-Schnelltestpartnersuche und Corona-Warn-App Version 2.4 mit digitalem Testzertifikat</a>",
+                            "<b>Wird bei 2G+ meine Auffrischimpfung berücksichtigt?</b>",
+                            "Laut Beschluss der Gesundheitsminister*innen von Bund und Ländern müssen Geimpfte oder Genesene, die eine Auffrischimpfung (Boosterimpfung) erhalten haben, bei einer 2G+-Regelung momentan keinen aktuellen Test vorlegen. Die Corona-Warn-App berücksichtigt diese Regelung in der Statusanzeige derzeit noch nicht. Bei vorhandenem Impf- oder Genesenenzertifikat und Zertifikat der Auffrischimpfung zeigt sie nicht den Status 2G+ an, sondern 2G. Das Projektteam arbeitet bereits daran, dass die Corona-Warn-App in Zukunft Zertifikate von Auffrischimpfungen entsprechend einbezieht.",

```

Important thing: Should be **bold**. 🙂

```suggestion
                            "Laut Beschluss der Gesundheitsminister*innen von Bund und Ländern müssen Geimpfte oder Genesene, die eine Auffrischimpfung (Boosterimpfung) erhalten haben, bei einer 2G+-Regelung momentan keinen aktuellen Test vorlegen. <b>Die Corona-Warn-App berücksichtigt diese Regelung in der Statusanzeige derzeit noch nicht</b>. Bei vorhandenem Impf- oder Genesenenzertifikat und Zertifikat der Auffrischimpfung zeigt sie nicht den Status 2G+ an, sondern 2G. Das Projektteam arbeitet bereits daran, dass die Corona-Warn-App in Zukunft Zertifikate von Auffrischimpfungen entsprechend einbezieht.",
```